README.md

```diff
@@ -116,6 +116,7 @@ Terminology:
 * [fman](https://fman.io/) - A dual-pane file manager.
   * :heavy_plus_sign: [VimNavigation](https://github.com/raguay/VimNavigation) - Adds Vim-style keyboard navigation to the fman file manager.
 * :white_check_mark: [yazi](https://yazi-rs.github.io/) - Blazing fast terminal file manager written in Rust, based on async I/O.
+* :white_check_mark: [superfile](https://superfile.dev/) - Pretty fancy and modern terminal file manager 
```

Comment:
I saw they have a help page with more Vim-like configuration. It would be great to link like this:

```suggestion
* :white_check_mark: [superfile](https://superfile.dev/) - Pretty fancy and modern terminal file manager that supports h-j-k-l out of the box. See also more vim-like configuration [here](https://superfile.dev/configure/custom-hotkeys/#vim-like-superfile-hotkeys).
```